docs/proxy/google_serp_proxy/nodejs_examples.md

```diff
@@ -191,46 +181,51 @@ Get a list of shopping results from the Czech Republic for keyword `Apple iPhone
         }
     });
 
-## [](#usage-with-request-promise) Usage with [request-promise](https://www.npmjs.com/package/request-promise) and [cheerio](https://www.npmjs.com/package/cheerio) NPM packages
+## [](#usage-with-request-promise) Usage with our [requestAsBrowser](https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions) and [cheerio](https://www.npmjs.com/package/cheerio) NPM package
```

Comment:
I'd change the anchor to `#usage-with-request`

Reply:
Changed.